### PR TITLE
feat: genome data portal app proof of concept

### PR DESCRIPTION
**Pull request by mswertz**
## Genome Portal App Demo

The purpose of this Vue application is to create a user interface(s) for finding data in the GDI portal (i.e., Fair Data Hub). Ideally, we would like users to search for data using any number of filters, and then be able to request access to the data (via REMs). For the proof of concept, we would like to demonstrate the search capabilities in EMX2 using builtin features and Beacon.

The app will include the following pages:

1. **Find Records**: a search interface that allows users to find data at the record-level (but only view aggregates)
2. **Find Datasets**: search interface that displays all datasets (i.e., collections of records) where users can select datasets of interest.

### CODE REVIEW

File: apps/molgenis-components/src/components/forms/InputRefList.vue
Comment (davidruvolo51) [inline apps/molgenis-components/src/components/forms/InputRefList.vue:1]: This shouldn't be committed.

Comment (davidruvolo51) [inline apps/molgenis-components/src/components/forms/InputRefList.vue:1]: This should not be committed until further discussion

**Review by davidruvolo51 — COMMENTED** [review]
The major issue is the `hackalldata` emit. This shouldn't be merged with the main branch. This should be further developed/discussed.

File: apps/gportal/src/components/Datasets.vue [hunk 1]
```diff
@@ -77,60 +65,31 @@ function setRefs(value) {
...
+function selectAll() {
+  checkboxes.value.forEach((value) => {
     if (selection.value.indexOf(value) === -1) {
       selection.value.push(value);
     }
   });
   setUrl();
 }
 
+onMounted(() => {
```
Comment (davidruvolo51) [inline apps/gportal/src/components/Datasets.vue:77]: This is really bad hack. Is there a slot or prop available to pass a column header?

Comment (mswertz) [reply apps/gportal/src/components/Datasets.vue:77]: not yet but added one

**Comment by mswertz** [comment]
@joerivandervelde I processed the review comments, can you check if it is still functional?

**Review by joerivandervelde — COMMENTED** [review]
Looks good, everything seems to work OK

**Review by joerivandervelde — APPROVED** [review]

**Pull request merged**